File: app/src/main/java/com/au/lyber/models/WithdrawAddress.java
package com.au.lyber.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class WithdrawAddress {

@SerializedName("network")
@Expose
public String network;
@SerializedName("address")
@Expose
public String address;
@SerializedName("name")
@Expose
public String name;
@SerializedName("origin")
@Expose
public String origin;
@SerializedName("creationDate")
@Expose
public String creationDate;

}

File: app/src/main/java/com/au/lyber/models/WithdrawAddress.kt
package com.au.lyber.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class WithdrawAddress {
    @SerializedName("network")
    @Expose
    var network: String? = null

    @SerializedName("address")
    @Expose
    var address: String? = null

    @SerializedName("name")
    @Expose
    var name: String? = null

    @SerializedName("origin")
    @Expose
    var origin: String? = null

    @SerializedName("creationDate")
    @Expose
    var creationDate: String? = null

    @SerializedName("exchange")
    @Expose
    var exchange: String? = null
}